INSTALL

```diff
@@ -1,13 +1,14 @@
 ===== HERBSTLUFTWM =====
 
-Copyright 2011-2013 Thorsten Wißmann. All rights reserved.
+Copyright Thorsten Wißmann and contributors. All rights reserved.
 
 This software is licensed under the "Simplified BSD License".
 See LICENSE for details.
 
 ==== Requirements ====
 Build dependencies:
-    - build-environment (gcc/other compiler, make)
+    - build environment (e.g. gcc, clang; make, ninja)
```

Comment:
Does the `;` have a meaning or is it a typo? Or do you want to express something like:
```
   (e.g. gcc or clang, make or ninja)
```

Reply:
I think you can just say "make" here because it's the default generator in CMake. I'd expect anybody that knows of ninja, to just try if it works instead of assuming otherwise just because it's not mentioned her.